ulb.py

```diff
@@ -123,7 +118,7 @@ class Data(ct.Structure):
 
 # Compile & attach bpf program
 print("\nCompiling & attaching bpf code ...")
-b = BPF(src_file ="ulb.c", debug=debug, cflags=["-Wno-incompatible-pointer-types", "-Wno-compare-distinct-pointer-types"])
+b = BPF(src_file ="ulb.c", debug=debug)
```

Comment:
Feature request: let the user to specify cflags using a CFLAGS environment variable.

Reply:
I create a new issue about that : #19.
Do not hesitate to create new issue for feature request.